### PR TITLE
Add proper logic to conditionally show widget assets

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [TEC-3650]

This PR adds the proper logic for conditionally showing the assets for events list widget.

**Artifact:**

image

[TEC-3650]: https://moderntribe.atlassian.net/browse/TEC-3650

### CODE REVIEW

File: src/Tribe/Views/V2/Hooks.php [hunk 1]
```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
 		$assets->disable_v1();
 	}
 
+	public function action_register_widget_styles() {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:151]: :no_entry_sign: Missing doc comment for function action_register_widget_styles()

<sub>hash: 86962694dda0b1b41931f8cc3e439cba</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 2]
```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
...
+		$style_dir = plugin_dir_url( TRIBE_EVENTS_FILE ) . 'src/resources/css/';
+
+		wp_register_style(
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:154]: :no_entry_sign: Resource version not set in call to wp_register_style(). This means new versions of the style will not always be loaded due to browser caching.

<sub>hash: daaead05c9b5b1da66057028d5074fac</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 3]
```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
...
+			'tribe-events-widgets-v2-skeleton',
+			$style_dir . 'widgets-skeleton.css',
+			[ 'tribe-common-skeleton-style' ],
+		);
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:158]: :no_entry_sign: PHP syntax error: syntax error, unexpected ')'

<sub>hash: 4f451f59c0000bdd9312c059a2845345</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 4]
```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
...
+
+		wp_register_style(
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:160]: :no_entry_sign: Resource version not set in call to wp_register_style(). This means new versions of the style will not always be loaded due to browser caching.

<sub>hash: bd355879d3de32e83b98c5dd340cfd0b</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 5]
```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
...
+			'tribe-events-widgets-v2-full',
+			$style_dir  . 'widgets-full.css',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:162]: :no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: 849b337279e60425df30c92fe3c0032c</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 5 error(s)

**Comment by Camwyn** [comment]
Screenshots for asset work:
http://p.tri.be/9EgRvb (styles with widget on screen) http://p.tri.be/Ns9mAe (scripts, same)
http://p.tri.be/gr9orT (calendar page, no widget)

File: src/Tribe/Views/V2/Widgets/Widget_List.php [hunk 6]
```diff
@@ -72,6 +73,56 @@ class Widget_List extends Widget_Abstract {
 		],
 	];
 
+	/**
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Widgets/Widget_List.php:76]: * :no_entry_sign: Doc comment for parameter "$context" missing
* :no_entry_sign: Doc comment for parameter "$view" missing

<sub>hash: c4ef70200ceaf627978404b6cdff6eb9</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

**Review by Camwyn — APPROVED** [review]

**Pull request merged**